pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbinstance/alloydbinstance_with_observability/_http.log

```diff
@@ -1102,23 +869,16 @@ X-Xss-Protection: 0
     "machineConfig": {
       "cpuCount": 2
     },
-    "maintenanceVersionName": "POSTGRES_16.20250910.01_18",
     "name": "projects/${projectId}/locations/europe-west1/clusters/alloydbcluster-${uniqueId}/instances/alloydbinstance-${uniqueId}",
     "nodes": [
       {
-        "zoneId": "europe-west1-d"
+        "zoneId": "europe-west1-c"
       }
     ],
     "observabilityConfig": {
       "assistiveExperiencesEnabled": false,
       "enabled": false,
-      "maxQueryStringLength": 10240,
```

Comment:
Looks like the observability config related logic is not all supported in the mock. Could you add them?